src/InMemoryCache.ts

```diff
@@ -4,20 +4,24 @@ const STORE_EXPIRATION_CHECK_IN_MS = 10;
 const DEFAULT_CACHE_TIME_IN_MS = 10;
 
 export default class InMemoryCache implements ICacheEngine {
-  private store: { [key: string]: string|number|boolean } = {};
+  private store: { [key: string]: string } = {};
   private storeCacheExpiration: { [key: string]: number } = {};
 
   constructor() {
     this.monitorStoreForExpiredValues();
   }
 
-  get(key: string): string|number|boolean {
```

Comment:
Check w/ team: is there use cases for number and boolean?

My thought: everything should be serialized to a string